app/models/certificate.server.ts

```diff
@@ -0,0 +1,64 @@
+import type { Certificate } from '@prisma/client';
+
+import { prisma } from '~/db.server';
+
+export type { Certificate } from '@prisma/client';
+
+export async function getCertificateByUsername(username: Certificate['username']) {
+  return prisma.certificate.findUnique({ where: { username } });
+}
+
+export async function createCertificate(
+  username: Certificate['username'],
+  subject: Certificate['subject'],
+  certificate: Certificate['certificate'],
+  orderUrl: Certificate['orderUrl'],
+  privateKey: Certificate['privateKey']
+) {
+  // Set expiration date 90 days from now
+  let validTo = new Date();
+  validTo.setDate(validTo.getDate() + 90);
```

Comment:
@dadolhay should we consider using some date library instead of native dates? We could use `date-fns` or `moment` or whatever you like (I am pro `date-fns`)

Reply:
I have not used `date-fns` before, but there is a library called `luxon`. It's from the same folks who created `moment`, and learning from the mistakes they created `luxon`. Just putting it on your radar, not saying that we have to use that or anything.

`moment` is a huge, heavy weight library, I'd not use it if I didn't have to.
To be honest, I'd not pull in a library until we have to, or it is clear that in the future we will need to do so. (so we can be consistent later)